app/controllers/book_suggestions_controller.rb

```diff
@@ -0,0 +1,6 @@
+class BookSuggestionsController < ApiController
+  skip_before_action :authenticate_request
```

Comment:
And, since ApplicationController doesn't have authentication, probably you can remove this line